patches/@ecl/twig-component-file+3.7.0.patch

```diff
@@ -0,0 +1,265 @@
+-      <div class="ecl-file__detail-meta">
+-        {{- _meta_detail_string -}}
++      {% for _label in _labels %}
++      <div class="ecl-file__label">
++        {% include '@ecl/label/label.html.twig' with _label only %}
++        {% endfor %}
++        {% endif %}
++        {% if  _meta_detail_string is not empty %}
++          <div class="ecl-file__detail-meta">
++            {{- _meta_detail_string -}}
++          </div>
++        {% endif %}
++        {% if _title is not empty %}
++          <div class="ecl-file__title">{{ _title }}</div>
++        {% endif %}
++        {% if _language is not empty %}
++          <div class="ecl-file__language">{{ _language }}</div>
++        {% endif %}
++        {% if _meta is not empty %}
++          <div class="ecl-file__meta">{{ _meta }}</div>
++        {% endif %}
++        {% else %}
++          {% if _language is not empty %}
++            <div class="ecl-file__language">{{ _language }}</div>
++          {% endif %}
++          {% if _meta is not empty %}
++            <div class="ecl-file__meta">{{ _meta }}</div>
++          {% endif %}
++        {% endif %}
+       </div>
+       {% endif %}
+-      {% if _title is not empty %}
+-      <div class="ecl-file__title">{{ _title }}</div>
+-      {% endif %}
+-      {% if _language is not empty %}
+-      <div class="ecl-file__language">{{ _language }}</div>
+-      {% endif %}
+-      {% if _meta is not empty %}
+-      <div class="ecl-file__meta">{{ _meta }}</div>
+-      {% endif %}
+-    {% else %}
+-      {% if _language is not empty %}
+-      <div class="ecl-file__language">{{ _language }}</div>
+-      {% endif %}
+-      {% if _meta is not empty %}
+-      <div class="ecl-file__meta">{{ _meta }}</div>
+-      {% endif %}
+-    {% endif %}
++      {% include '@ecl/link/link.html.twig' with _download|merge({
++        link: _download.link|default({})|merge({
++          type: 'standalone',
++          icon_position: 'after',
++        }),
++        icon: _download.icon|default({})|merge({
++          name: 'download',
++          size: 'fluid'
++        }),
++        extra_classes: 'ecl-file__download',
++      }) only %}
+     </div>
+-  {% endif %}
+-  {% include '@ecl/link/link.html.twig' with _download|merge({
+-    link: _download.link|default({})|merge({
+-      type: 'standalone',
+-      icon_position: 'after',
+-    }),
+-    icon: _download.icon|default({})|merge({
+-      name: 'download',
+-      size: 'fluid'
+-    }),
+-    extra_classes: 'ecl-file__download',
+-  }) only %}
++    {% if _translation is not empty %}
++      {% include '@ecl/file/file-translations.html.twig' with _translation|merge({
++        variant: _variant
++      }) only %}
++    {% endif %}
+   </div>
+-{% if _translation is not empty %}
+-  {% include '@ecl/file/file-translations.html.twig' with _translation|merge({
+-    variant: _variant
+-  }) only %}
+-{% endif %}
+-</div>
+ 
+-{% endapply %}
++  {% endapply %}
```

Comment:
It would be nice to leave only real changes without change only in indentation otherwise it is hard to review and maintain this patch in the future.

Reply:
Fixed.